core/clients/key_flow.go

```diff
@@ -200,19 +202,15 @@ func (c *KeyFlow) configureHTTPClient() {
 }
 
 // validate the client is configured well
-func (c *KeyFlow) validate() error {
-	if c.config.ServiceAccountKey == "" {
+func (c *KeyFlow) validate() (err error) {
```

Comment:
Why this change? It's unneccessary, doesn't hurt anything, seems just random

Reply:
It was necessary for variable assignment, but I updated it to keep the function signature